ReverseDSC.Core.psm1

```diff
@@ -769,8 +769,12 @@ we should not have commas in between items it contains.
     if ($IsCIMArray)
     {
         $DSCBlock = $DSCBlock.Replace("},`r`n", "`}`r`n")
-        #$DSCBlock = $DSCBlock.Replace("}`r`n,", "`}`r`n")  # see bbelow
+        #$DSCBlock = $DSCBlock.Replace("}`r`n,", "`}`r`n") # see below
         $DSCBlock = $DSCBlock -replace "\}`r`n\s*,", "}`r`n," # replace "}<crlf>[<whitespace>]," with "}<crlf>"
+
+        $DSCBlock = $DSCBlock.replace("    ,`r`n", "    `r`n")
```

Comment:
I am a little concerned about this one. Why would we be removing commas? I feel like this would risk breaking certain CIMInstance Arrays....

Reply:
@William-Francillette Do you have any concerts about this replace in particular? I honestly didn't see any exports where this would happen, the only one I've seen so far was the below and that is covered in this PR.

```powershell
"`r`n,`r`n"
```

Reply:
I may have encountered the case, but it would be on an empty line so we could replace it by
$DSCBlock = $DSCBlock.replace("`r`n    ,`r`n", "`r`n")